app/grants/templates/grants/index.html

```diff
@@ -174,6 +170,25 @@
         document.sub_round_slug = "{{clr_round.sub_round_slug}}";
         document.customer_name = "{{clr_round.customer_name}}";
       {% endif %}
+      let base_style = {
+          bg: '/static/v2/images/bg/{{styles.bg}}',
+          size: "{{styles.bg_size}}",
+          color: "{{styles.bg_color}}",
+          background_image: "url('/static/v2/images/bg/{{bottom_back}}')"
+        };
+
+      document.all_type_styles = {{ all_styles|safe }}
+      document.all_routing_policies = {{ all_routing_policies | safe}}
+      document.all_routing_policies.push(base_style)
+      {% if grant_bg %}
+        document.current_style = {
+          banner_image: "{{ grant_bg.banner_image }}",
+          background_image: " url('{{background_image }}')"
+        }
+      {% else %}
+        document.current_style = base_style
+      {% endif %}
```

Comment:
^ this section would alos apply for the CLR pages cause they use the same code which is not need right ? I think it's okay but just wanted to check

Reply:
I don't fully comprehend this comment.

Reply:
oh nvm ! mis-understood this